opencv/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: opencv
+  version: 2.4.11
+
+source:
+  fn: opencv-2.4.11.zip
+  url: http://sourceforge.net/projects/opencvlibrary/files/opencv-unix/2.4.11/opencv-2.4.11.zip/download
+  sha1: d6e3048416d42213c204f89b9dfe39742f9a708c
+
+  patches:
+   - osx_rpath.patch  # [osx]
+
+build:
+  number: 0
```

Comment:
Adding the following would ensure Python 2.7 builds only.

    skip: true  # [not py27]